### DISCUSSION TITLE
Add constant components to subject DN

### DISCUSSION THREAD

**Discussion by hansjoachimknobloch** [Sleepw4lker/TameMyCerts#11] [Ideas]
Hi Uwe,

from what I understand, TameMyCerts can currently build a subject DN from AD attributes using
<SubjectDistinguishedName> <RelativeDistinguishedName> <Field/> <DirectoryServicesAttribute>
tags.

In a current project have a requirement/desire to add a constant DN suffix (e.g. "O=MyCompany,L=MyCity,SP=MyState,C=DE") to TLS server certificates. That might be done e.g. by introducing a new tag <ConstantValue> to the subject rules (in <Subject> <SubjectRule <Field/> ,,,).

Do you already have something like this functionality on your todo list? Otherwise I might try my luck adding that extension.

Best regards,
  Hans-Jochachim.

**Comment by Sleepw4lker** [comment]
Hello @hansjoachimknobloch yes I have that on my backlog but with no concrete schedule. Maybe we should work together working out the details. Cheers

**Comment by Sleepw4lker** [comment]
Closing, as this has been implemented since v1.6
